src/frontend/src/pages/CreateChangeRequestPage/CreateChangeRequestView.tsx

```diff
@@ -96,6 +96,12 @@ const CreateChangeRequestsView: React.FC<CreateChangeRequestViewProps> = ({
           why: [{ type: ChangeRequestReason.Other, explain: 'The cost of materials ended up exceeding the initial budget' }],
           type: ChangeRequestType.Issue
         }
+      : query.get('createWP')
+      ? {
+          what: crDesc,
+          why: [{ type: ChangeRequestReason.Initialization, explain: 'Creating a Work Package on this Project' }],
+          type: ChangeRequestType.Issue
```

Comment:
we should probably set this to a Definition Change on second thought

Reply:
Should be fixed now